clipper_admin/clipper_admin/kubernetes/kubernetes_container_manager.py

```diff
@@ -370,6 +377,7 @@ def _start_prometheus(self):
                 CONFIG_FILES['metric']['deployment'],
                 version=PROM_VERSION,
                 cluster_name=self.cluster_name,
+                service_account_name=self.cluster_name+"-prometheus"
```

Comment:
I can't find `service_account_name` in prom_deployment.yaml.

Reply:
So, I decided to use `default` as prom_deployment service account because 
1. it works 
2. There's no reason to setup specific service account for /metrics for users. We can just leave it to users. 

I will delete this line.